app/src/main/res/values/strings.xml

```diff
@@ -5,6 +5,64 @@
     <string name="tomorrow_icon">Tomorrow</string>
     <string name="search_icon">Search</string>
     <string name="home_fragment">HOME!</string>
-    <!-- TODO: Remove or change this placeholder text -->
-    <string name="hello_blank_fragment">Hello blank fragment</string>
+    <string name="city">Palermo,Sicily</string>
+    <string name="today">Today,</string>
+    <string name="today_date">Sunday 10 september</string>
+    <string name="hour_now">11:00</string>
+    <string name="grade_now">31°</string>
+    <string name="humidity_now">0%</string>
```

Comment:
The idea is not to have the data that belong to the data class into the strings file. It doesn't make sense.